.github/workflows/pyrealm_ci.yaml

```diff
@@ -48,12 +48,12 @@ jobs:
       env:
         CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
 
-    - name: Setup Git LFS
-      id: lfs
-      if: ${{ steps.codecov.outcome == 'success' && github.event_name != 'push' }}
-      run: |
-        git lfs pull
-        git lfs ls-files -n | xargs git lfs track  # track all LFS files
+    # - name: Setup Git LFS
+    #   id: lfs
+    #   if: ${{ steps.codecov.outcome == 'success' && github.event_name != 'push' }}
+    #   run: |
+    #     git lfs pull
+    #     git lfs ls-files -n | xargs git lfs track  # track all LFS files
```

Comment:
Can you delete rather than comment out? I used to comment retired options too - so that I had a reminder of how to do them - but we have this in the git history if we need it and keeping the files clean is better practice.

Reply:
I have deleted it now.